openedx/stanford/cms/envs/common.py

```diff
@@ -10,6 +10,9 @@
 
 STANFORD_ROOT = REPO_ROOT / 'openedx/stanford'
 
+CELERY_IMPORTS = (
```

Comment:
I think this will overwrite the value in lms/envs/common. Shouldn't you be appending to CELERY_IMPORTS?

Reply:
@dcadams Ah, this file imports from `lms/envs/common.py` which does define it;
I'll update.

Reply:
Actually, upon further inspection, this variable is _not_ imported into the CMS file.

Attempting to update the value in stanford's `common.py` results in:
> NameError: name 'CELERY_IMPORTS' is not defined

So we do want to define it out-right.

Reply:
@dcadams BTW: I tried with:

```py
CELERY_IMPORTS = CELERY_IMPORTS + (
    'openedx.stanford.cms.djangoapps.contentstore.views.utilities.tasks',
)
```